internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -357,11 +357,11 @@ func (provisioner *KopsProvisioner) ProvisionCluster(cluster *model.Cluster, aws
 			logger.Infof("Waiting up to %d seconds for %q pod %q to start...", wait, deployment, pod.GetName())
 			ctx, cancel := context.WithTimeout(context.Background(), time.Duration(wait)*time.Second)
 			defer cancel()
-			pod, err := k8sClient.WaitForPodRunning(ctx, namespace, pod.GetName())
+			_, err := k8sClient.WaitForPodRunning(ctx, namespace, pod.GetName())
 			if err != nil {
 				return err
 			}
-			logger.Infof("Successfully deployed operator pod %q", pod.Name)
+			logger.Infof("Successfully deployed service pod %q", pod.Name)
```

Comment:
Now that pod is removed from line 360, does pod.Name still get a value?

Reply:
the value comes from this `for _, pod := range pods.Items {`

Reply:
It shouldn't be pod.GetName()?

Reply:
fixed